test/utils.rb

```diff
@@ -220,11 +220,15 @@ def start_server(verify_mode: OpenSSL::SSL::VERIFY_NONE, start_immediately: true
                 readable, = IO.select([ssls, stop_pipe_r])
                 break if readable.include? stop_pipe_r
                 ssl = ssls.accept
-              rescue OpenSSL::SSL::SSLError, IOError, Errno::EBADF, Errno::EINVAL,
+              rescue OpenSSL::SSL::SSLError => e
+                retry if ignore_listener_error
+                raise unless e.message =~ /inappropriate fallback/
+              rescue IOError, Errno::EBADF, Errno::EINVAL,
```

Comment:
Are changes in start_server really needed? I'd like to have all SCSV specific code in TestSSL#test_fallback_scsv.

Reply:
An exception occurs on **server** part too in case of SCSV error.
Without this catch, unit test detect only the server part and not the client part as expected.
I didn't find a proper way to do this :'(

Reply:
You can get a raw TCP socket pair by `socketpair` defined in TestSSL when the exception raised by SSLSocket#accept has to be tested.

Reply:
Ah, I misinterpreted your comment. Errors raised in server can be suppressed by passing 'ignore_listener_error: true' to start_server.

Reply:
I fix the test with `socketpair`, to also test server side error :)